google/cloud/logging_v2/handlers/handlers.py

```diff
@@ -206,9 +203,8 @@ def emit(self, record):
         labels = record._labels
         message = _format_and_parse_message(record, self)
 
-        if resource.type == _GAE_RESOURCE_TYPE and record._trace is not None:
-            # add GAE-specific label
-            labels = {_GAE_TRACE_ID_LABEL: record._trace, **(labels or {})}
+        labels = {**add_environment_labels(resource, record), **(labels or {})} or None
```

Comment:
Just to double-check: That this is only for the CloudLoggingHandler, not the StructuredLogHandler. That's intended, right?